decidim-meetings/spec/controllers/decidim/meetings/meetings_controller_spec.rb

```diff
@@ -101,7 +101,7 @@
         expect(flash[:alert]).to be_blank
       end
 
-      it "can NOT access private and non transparent meetings" do
+      it "CANNOT access private and non transparent meetings" do
```

Comment:
let's use the lower case here ... 
https://github.com/decidim/decidim/pull/10551#discussion_r1136672685